src/components/structures/RoomView.js

```diff
@@ -74,6 +75,8 @@ module.exports = React.createClass({
             syncState: MatrixClientPeg.get().getSyncState(),
             hasUnsentMessages: this._hasUnsentMessages(room),
             callState: null,
+            readReceiptEventId: room.getEventReadUpTo(MatrixClientPeg.get().credentials.userId),
+            readMarkerGhostEventId: undefined,
```

Comment:
Actually this is probably just that readReceiptEventId should be readMarkerEventId.